collect_app/build.gradle

```diff
@@ -320,6 +320,8 @@ dependencies {
     // Better "Subjects" for Rx:
     implementation "com.jakewharton.rxrelay2:rxrelay:2.1.1"
 
+    implementation 'com.github.Armen101:AudioRecordView:1.0.2'
```

Comment:
This was the simplest waveform library I could find that still looked supported. I think we may want to improve this in the future or add our own flair.

Reply:
I see a size increase of .2mb from 1.28.4 to this branch. Not sure how much is from this lib but either way it seems ok.